### DISCUSSION TITLE
SORT_RO support

### DISCUSSION THREAD

**Discussion by emilienkofman** [redis/go-redis#2352] [Ideas]
Hi and thank you for the great library.

I've figured out redis v7 supports a read only version of the SORT command that can thus be used on a read-only slave replica, see [SORT](https://redis.io/commands/sort/) and [SORT_RO](https://redis.io/commands/sort_ro/). I could not find SORT_RO in the v9 library (maybe I missed it?). If I want to add it would you suggest adding a sort option [here](https://github.com/go-redis/redis/blob/master/commands.go#L705) or making it a separate command to stick to the redis documentation?

Sincerely

**Comment by vmihailenco** [comment]
Hello,

I would add `SORT_RO` as a separate command, but I am open to suggestions.

**Reply by emilienkofman** [reply]
ok thanks for the advice, I've opened a pull request here https://github.com/go-redis/redis/pull/2354
